src/uwtools/tests/config/formats/test_sh.py

```diff
@@ -40,7 +41,20 @@ def test_parse_include():
     assert len(cfgobj) == 5
 
 
-def test_sh(salad_base):
+@mark.parametrize("func", [repr, str])
+def test_sh_repr_str(func):
+    config = fixture_path("simple.sh")
+    expected = """
+    base=kale
+    fruit=banana
+    vegetable=tomato
+    how_many=12
+    dressing=balsamic
+    """
+    assert func(SHConfig(config)) == dedent(expected).strip()
+
+
+def test_sh_sh(salad_base):
```

Comment:
Based on the other formatting tests, I think you wanted to keep the name of this test what it was previously called.

```suggestion
def test_sh(salad_base):
```